requirements.txt

```diff
@@ -1,8 +1,8 @@
 psutil>=5.7.2
 coloredlogs==10.0
 pandas>=1.1.3
-redis==3.0.1
-redis-py-cluster==2.0.0
+redis==3.5.3
```

Comment:
Will these be needed for Ray? We are planning on taking them out with #14 

Also, the smartsim ray functionaility should have it's own dependency section)

```cfg
[options.extras_require]
dev =
    black>=20.8b1
    torchvision==0.8.2
    torch==1.7.1
    isort>=5.6.4
    pylint>=2.6.0
    matplotlib>=3.3.3
    pytest>=6.0.0
    pytest-cov>=2.10.1

doc=
    sphinx==3.1.1
    breathe==4.27.0
    sphinx_rtd_theme>=0.5.0
    sphinx-fortran==1.1.1
    nbsphinx>=0.8.2

ray=
 .....
.......
```

The ray section will just be the needed requirements to run Ray workloads.

users will install with ``pip install smartsim[ray]`` or ``pip install smartsim[dev,ray]`` which will also install ray extra dependencies.

Reply:
I had to update the version numbers because they were conflicting with what Ray required. If we take them out, Ray will install what it needs and we will not have to care about versions anymore, which is good.

Reply:
Adapted the requirements to those currently in `develop`, created `ray` section in `setup.cfg` (with only Ray, right now).